app/models/product.rb

```diff
@@ -0,0 +1,5 @@
+class Product < ApplicationRecord
+  has_many :order_products
+  has_many :order through: :order_products
```

Comment:
```suggestion
  has_many :orders through: :order_products
```
theo lí thuyết thì phải là thế này chứ